blueprints/app/files/app/index.html

```diff
@@ -15,6 +15,10 @@
     {{content-for "head-footer"}}
   </head>
   <body>
+    <noscript>
+      JavaScript must be enabled to run this application.
```

Comment:
I must be misunderstanding how Fastboot works. Does it not require putting checks before any DOM operations in hooks like `didInsertElement`? This can't be avoided in new or old apps, so I'm not seeing Fastboot as being "free", thus I was saying that it requires work on the part of the developer to keep the app Fastboot "compliant".

> it is going to be confusing for the community.

Sounds like an education problem. But at the same time, if it was wrapped in something like:

```hbs
{{#unless isFastboot}}
  <noscript>...</noscript>
{{/unless}}
```

The confusion is minimized since it's obvious what happens in the Fastboot scenario.

---

I'm fine to remove this if it really is a bad idea. Just want to make sure we are
talking about the same things first.